doto-frontend/src/components/MarketPlace.js

```diff
@@ -0,0 +1,134 @@
+/* @author: Utsav Trivedi */
```

Comment:
I dont think we need authors on the files, If people are interested, they can git blame or similar.